index.js

```diff
@@ -19,10 +19,10 @@ module.exports = {
 		function build( data ) {
 			data.parts.push(
 				'<head>' +
-					'<script src="' + path.join( '/plugins', sinonPath ) + '"></script>' +
+					'<script src="/plugins/' + sinonPath.split( path.sep ).join( '/' ) + '"></script>' +
```

Comment:
I'm afraid we still have to build the path using `path.join()` because of UNIX paths starting with `/`. 
So I'd still go with:

```js
'<script src="' + path.join('/plugins/', sinonPath).split( path.sep ).join( '/' ) + '"></script>' +
```
Sorry... :wink: